contrib/containers/ci/common/Dockerfile.finish

```diff
@@ -0,0 +1,11 @@
+RUN mkdir /dash-src && \
```

Comment:
Comment that explicitly states that this file should be last in the Dockerfile chain would improve documentation